code/__HELPERS/unsorted.dm

```diff
@@ -1507,3 +1507,10 @@ proc/pick_closest_path(value, list/matches = get_fancy_list_of_atom_types())
 	else if(powerused < 1000000000) //Less than a GW
 		return "[round((powerused * 0.000001),0.001)] MW"
 	return "[round((powerused * 0.000000001),0.0001)] GW"
+
+/proc/living_player_count()
+	var/living_player_count = 0
+	for(var/mob in GLOB.player_list)
+		if(mob in GLOB.alive_mob_list)
```

Comment:
Wouldn't it be less resource intensive to check the stat, as that's a simple number rather than looping through the entire list?

Reply:
This is bad code alright, but this is bad /tg/ code at least, and LetterN won't be fixing it judging by how long this PR is taking.
I'd say accept this due to /tg/ compatibility and let someone more wiling and able fix it later.